plugins/plugin_utils/hashi_vault_plugin.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021 Brian Scholer (@briantist)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+from ansible.plugins import AnsiblePlugin
+from ansible import constants as C
+from ansible.utils.display import Display
+
+from ansible_collections.community.hashi_vault.plugins.module_utils.hashi_vault_common import HashiVaultHelper
+
+display = Display()
+
+
+class HashiVaultPlugin(AnsiblePlugin):
+    def __init__(self, loader=None, templar=None, **kwargs):
+        super(HashiVaultPlugin, self).__init__()
+
+        self.helper = HashiVaultHelper()
+
+    def deprecate(self, collection_name='community.hashi_vault'):
```

Comment:
How about calling this `process_deprecations` instead? Then you don't need the pylint ignore when calling this :)

Reply:
I will give it a try. I already tried adding `_` and `__` and pylint still hit it, I'll see if a more different name fixes that. It's weird because it was called `deprecate` when it was part of `hashi_vault` and yet never caused a lint problem.